process-engine/skeleton
# 👷 Add performance-tests with Jenkins

## What did you change?

I changed the Jenkinsfile so that it now runs performance-tests on every commit. The procedure is summarized [here](https://github.com/process-engine/skeleton/blob/403ed07552fbd13efedad8a5faf8a43f5d0c00b5/README.md#performance-tests).

![image](https://user-images.githubusercontent.com/20770029/36232632-e06b4914-11e2-11e8-8e5c-9273e4a92bf8.png)

## How can others test the changes?

Open a feature-branch on this repo, merge this feature branch into yours and see how Jenkins starts performance-testing it!

## PR-Checklist

Please check the boxes in this list after submitting your PR:
